frontend/static/quotes/english.json

```
@@ -38794,6 +38794,12 @@
       "source": "Fred Rogers",
       "length": 213,
       "id": 6815
+    }
```

JSON objects need to end with a comma if it is not the last one.
```suggestion
    },
```